src/gui/properties/peerlistwidget.cpp

```diff
@@ -89,7 +89,7 @@ PeerListWidget::PeerListWidget(PropertiesWidget *parent)
     , m_properties(parent)
 {
     // Load settings
-    loadSettings();
+    const bool columnLoaded = loadSettings();
```

Comment:
I would rewrite it to be more explicit about the fact that we try to restore header state:
```suggestion
    const bool columnLoaded = header()->restoreState(Preferences::instance()->getPeerListState());
```

Reply:
> It was already discussed above [#17940 (review)](https://github.com/qbittorrent/qBittorrent/pull/17940#pullrequestreview-1161146242). Reviewer told me to do exactly the opposite.
> 
> > you could re-use `loadSettings()`

He just offered to reuse it. I propose to completely discard/remove `loadSettings()` wrapper, and use its contents directly, since it is trivial and it would look more clearly in the place of the call.

Reply:
Okay, forget it.